src/libinjector/linux/linux_debug.h

```diff
@@ -0,0 +1,10 @@
+#ifndef LINUX_DEBUG_H
+#define LINUX_DEBUG_H
+
+#include "linux_utils.h"
+
+void print_hex(char* shellcode, int len, int bytes_write_read);
+void print_stack(drakvuf_t drakvuf, drakvuf_trap_info_t* info);
+void print_registers(drakvuf_trap_info_t* info);
+
+#endif
```

Comment:
This file isn't linux related at all (not even injector related), I'd move it somewhere else. Same for implementation.

Reply:
where would you suggest it, and can you suggest some filename also please then?

Reply:
moved it to `libinjector/debug_helpers.[c,h]`